shared/src/Unison/ABT.hs

```diff
@@ -239,25 +272,33 @@ instance (Foldable f, Functor f, Eq1 f) => Eq (Term f) where
     go (Tm f1) (Tm f2) = f1 ==# f2
     go _ _ = False
 
-instance J.ToJSON1 f => ToJSON (Term f) where
```

Comment:
Now that we are using ghcjs, might we just auto derive these? It's one less parsing/serializing/marshaling nuisance to avoid -- might as well dogfood our own philosophy!